src/main/java/supertracker/parser/Parser.java

```diff
@@ -33,12 +37,17 @@ public class Parser {
     private static final String NAME_FLAG = "n";
     private static final String QUANTITY_FLAG = "q";
     private static final String PRICE_FLAG = "p";
-    private static final String SORT_QUANTITY_FLAG = "sq";
-    private static final String SORT_PRICE_FLAG = "sp";
-    private static final String REVERSE_FLAG = "r";
+    private static final String EX_DATE_FLAG = "e";
     private static final String NAME_GROUP = "name";
     private static final String QUANTITY_GROUP = "quantity";
     private static final String PRICE_GROUP = "price";
+    private static final String EX_DATE_GROUP = "expiry";
+    private static final String EX_DATE_FORMAT = "dd/MM/yyyy";
```

Comment:
I suggest using dashes instead of slashes to use as a date separator to avoid confusion with the parameter flags currently.
```suggestion
    private static final String EX_DATE_FORMAT = "dd-MM-yyyy";
```

Reply:
> I suggest using dashes instead of slashes to use as a date separator to avoid confusion with the parameter flags currently.

Thank you Ryan! I have implemented the changes.